src/content.css

```diff
@@ -1,8 +1,9 @@
 .gfi {
   color: rgba(3,47,98,0.55);
+  display: inline-block;
+  vertical-align: top;
 }
 
 td.icon > svg.octicon-file, .rgh-edit-files-faster {
   display: none;
 }
```

Comment:
According to your [new implementation of the icons](https://github.com/lvarayut/github-file-icons/pull/17/files#diff-95ffcbff3c841ad0a802793cabd8940eR9-R19), the icons will be shown inside the `.rgh-edit-files-faster`. Hence, the following code needs to be deleted in order to display the icons correctly:

```
td.icon > svg.octicon-file, .rgh-edit-files-faster {
 display: none;
}
```

Reply:
Ah, I was wondering why the icons didn't show up yesterday (I initially made this PR with Refined GitHub disabled so I didn't notice the issue). Nice catch!